scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/Task.java

```diff
@@ -380,6 +400,27 @@ public static Task parseJson(JSONObject jsonTask) {
         result.setTag(tag);
         result.setJobId(jobId);
         result.setJobName(jobName);
+
+        long startAtTime = 0L;
+        JSONObject genericInformationsObject = jsonTask.get("genericInformations").isObject();
+        if (genericInformationsObject != null) {
+            JSONValue genericInformationsValue = genericInformationsObject.get("START_AT");
+            if (genericInformationsValue != null) {
+                String genericInformationsStr = genericInformationsValue.isString().stringValue();
```

Comment:
What I suggested is only on Scheduler core. I think users should enter all their datetime using ISO 8601 format and then when it is received on the scheduler core, the value is transformed in the local timezone of the scheduler, optionally stored in this format and eventually used.

This is for the incoming side. For the outcoming side, the scheduler should send back a datetime in ISO 8601 format using its timezone and this datetime is then transformed on client side to datetime that satisfies user timezone.